### ISSUE TITLE
Incorrect `memmove` library function specification

### ISSUE THREAD

**Issue by sim642** [goblint/analyzer#1121]
Our old-style library function specifications for `memmove` are the following:
https://github.com/goblint/analyzer/blob/91388efe40f0ea7f432068bfa34c91cce00a82b1/src/analyses/libraryFunctions.ml#L764-L766
These seem to be flipped, because the actual signature is
```c
void* memmove( void* dest, const void* src, size_t count );
```
So the first argument is written and the second one read.

This causes some spurious race warning in chrony (due to the extra `src` write) but also could be unsound (due to the missing `dst` write).

This could also be done in #1079.
